tests/playwright/src/model/pages/create-kind-cluster-page.ts

```diff
@@ -0,0 +1,115 @@
+  readonly logsButton: Locator;
+  readonly providerTypeCombobox: Locator;
+  readonly httpPort: Locator;
+  readonly httpsPort: Locator;
+  readonly containerImage: Locator;
+
+  constructor(page: Page) {
+    super(page);
+    this.clusterPropertiesInformation = this.page.getByRole('form', { name: 'Properties Information' });
+    this.clusterNameField = this.clusterPropertiesInformation.getByRole('textbox', { name: 'Name', exact: true });
+    // Locator for the parent element of the ingress controller checkbox, used to change its value
+    this.controllerCheckbox = this.clusterPropertiesInformation
+      .getByRole('checkbox', {
+        name: 'Setup an ingress controller',
+      })
+      .locator('..');
+    this.clusterCreationButton = this.clusterPropertiesInformation.getByRole('button', { name: 'Create', exact: true });
+    this.logsButton = this.clusterPropertiesInformation.getByRole('button', { name: 'Show Logs' });
+    this.providerTypeCombobox = this.clusterPropertiesInformation.getByRole('combobox', { name: 'Provider Type' });
+    this.httpPort = this.clusterPropertiesInformation.getByLabel('HTTP Port');
+    this.httpsPort = this.clusterPropertiesInformation.getByLabel('HTTPS Port');
+    this.containerImage = this.clusterPropertiesInformation.getByPlaceholder('Leave empty for using latest.');
+    this.goBackButton = this.page.getByRole('button', { name: 'Go back to resources' });
+  }
+
+  public async createClusterDefault(clusterName: string, timeout?: number): Promise<void> {
+    await this.fillTextbox(this.clusterNameField, clusterName);
+    await playExpect(this.providerTypeCombobox).toHaveValue('podman');
+    await playExpect(this.httpPort).toHaveValue('9090');
+    await playExpect(this.httpsPort).toHaveValue('9443');
+    await playExpect(this.controllerCheckbox).toBeChecked();
+    await playExpect(this.containerImage).toBeEmpty();
+    await this.createCluster(timeout);
+  }
+
+  public async createClusterParametrized(
+    { clusterName, providerType, httpPort, httpsPort, useIngressController, containerImage }: KindClusterOptions = {},
+    timeout?: number,
+  ): Promise<void> {
+    if (clusterName) {
+      await this.fillTextbox(this.clusterNameField, clusterName);
+    }
+
+    if (providerType) {
+      await playExpect(this.providerTypeCombobox).toBeVisible();
+      const providerTypeOptions = await this.providerTypeCombobox.locator('option').allInnerTexts();
+      if (providerTypeOptions.includes(providerType)) {
+        await this.providerTypeCombobox.selectOption({ value: providerType });
+        await playExpect(this.providerTypeCombobox).toHaveValue(providerType);
+      } else {
+        throw new Error(`${providerType} doesn't exist`);
+      }
+
+      if (httpPort) {
+        await this.fillTextbox(this.httpPort, httpPort);
+      }
+      if (httpsPort) {
+        await this.fillTextbox(this.httpsPort, httpsPort);
+      }
+
+      if (!useIngressController) {
```

Comment:
I would add else branch since if the default setup will change, passing the `true` value wont change anything. so please also add else part.